site/js/server.js

```diff
@@ -0,0 +1,53 @@
+var Server = {
+    Automaton: {
+        create: function( data, callback, error ) {
+            $.post( 'api/automaton/create', {
+                name: '',
+                data: data
+            }, function( result ) {
+                callback( result );
+            }, 'json' ).error( function( jqXHR, settings, thrownError ) {
```

Comment:
Why not just `callback` instead of `function( result ) { callback( result ); }`?